monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -205,6 +217,19 @@ class ConfigurePageComponent extends AuthComponent {
   updateConfigSection = () => {
     let newConfig = this.state.configuration;
     if (Object.keys(this.currentFormData).length > 0) {
+
+      if (this.currentSection == 'monkey') {
+        let tempMatrix = this.state.attackConfig;
+        if (this.currentFormData['general']['post_breach_actions'].includes('BackdoorUser')) {
+          tempMatrix['persistence'].properties['T1136'].value = true;
+        }
+        else {
+          tempMatrix['persistence'].properties['T1136'].value = false;
+        }
+        this.setState({attackConfig: tempMatrix});
+        this.matrixSubmit();
+      }
```

Comment:
It's nice that you mapped it back to the matrix, but unnecessary. The config is more complicated/in depth than matrix, so we don't really do config -> matrix mapping, only matrix -> config. Attack matrix is a way to configure the configuration, if you will. You should remove this code, because we should either map every configuration value back to the attack technique (IDK if that's even possible) or map nothing at all (what we currently do).